.github/actions/detect_changes/action.yml

```diff
@@ -37,12 +37,21 @@ runs:
           echo "changes=false" >> $GITHUB_OUTPUT || \
           echo "changes=true" >> $GITHUB_OUTPUT
 
+      - name: Detect changes in seeding compared to main using git diff
+        id: detect_seeding_changes
+        if: github.ref != 'refs/heads/main'
+        shell: bash
+        run: |
+          git diff --quiet HEAD origin/main -- scripts/non_live_seeding && \
+          echo "changes=false" >> $GITHUB_OUTPUT || \
+          echo "changes=true" >> $GITHUB_OUTPUT
+
       - name: Determine if changes were detected
         id: determine_changes
         shell: bash
         run: |
-          if [[ "${{ steps.detect_changes.outputs.changes }}" == "true"  || "${{ steps.detect_shared_changes.outputs.changes }}" == "true" ]]; then
+          if [[ "${{ steps.detect_changes.outputs.changes }}" == "true"  || "${{ steps.detect_shared_changes.outputs.changes }}" == "true" || "${{ steps.detect_seeding_changes.changes }}" == "true" ]]; then
```

Comment:
does steps.detect_seeding_changes.changes need to be steps.detect_seeding_changes.outputs.changes?

Reply:
An alternative approach: because of how git diff works, I think we could also add an additional directory to inputs.path, e.g. in .github/workflows/docker_job.yml:

```
          - image_name: online-lpa/seeding_app
            dockerfile_path: ./service-seeding/docker/app
            service_path: ./service-seeding ./scripts/non_live_seeding
```

Even better, we could just move the scripts into service-seeding, as they aren't used for anything else afaik.

Reply:
I would probably just leave this who you've change it for now, though I think you might possibly need outputs as explained above.

Reply:
Ah thanks, yes it is supposed to be, I missed it somehow.

Moving the seeding scripts to service-seeding would be quite good actually, there's nothing else in that directory.